### PR TITLE
Added changes for the activity in files README.md, sl.c and sl.h

### PR DESCRIPTION
**Pull request by ArielGP**

### CODE REVIEW

File: sl.c [hunk 1]
```diff
@@ -112,7 +112,7 @@ void option(char *str)
 
 int main(int argc, char *argv[])
 {
-    int x, i, k;
+    int x, i, j; //Changed last variable form k to j
```
Comment (RaulSuarez1594) [inline sl.c]: I would like you to update the program and replace any occurrence of the x variable with a new variable named 'q'.

The new variable j can stay the same.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ A generic makefile for starting projects
 
     words here
 
+    Added words because i'm bored
```
Comment (RaulSuarez1594) [inline README.md:27]: Please add more words. 

:)

**Review by RaulSuarez1594 — CHANGES_REQUESTED** [review]

File: sl.c (hunk 1 above)
Comment (ArielGP) [reply sl.c]: The 'x' declaration on line 115 was changed to 'q' as requested

File: README.md (hunk 2 above)
Comment (ArielGP) [reply README.md:27]: Even more words added to the README.md file

**Review by RaulSuarez1594 — COMMENTED** [review]
Approved.

**Pull request merged**